### PR TITLE
Fix forms namespace

### PR DESCRIPTION
**Pull request by danut007ro**
## Fix forms namespace in form_types.xml

After moving the namespace from `Sonata\CoreBundle\Form` to `Sonata\Form` the form_types.xml and date.xml still pointed to old namespace.

I am targeting this branch, because it is a bug fix.

## Changelog

Fix namespace in form_types.xml and date.xml

### CODE REVIEW

**Comment by greg0ire** [comment]
Can you please test #610? I think it might solve your issue.

**Comment by danut007ro** [comment]
> Can you please test #610? I think it might solve your issue.

Sorry it didn't fixed it, and I found another issue where `use Sonata\CoreBundle\Model\Adapter\AdapterInterface;` in TwigExtension.php should be replaced with `use Sonata\Doctrine\Adapter\AdapterInterface;`.

I'm on php7.3 but this shouldn't matter I think.

**Comment by greg0ire** [comment]
There are some classes I could not touch, the base picker type related ones, are those the one you are still having issues with?

**Comment by danut007ro** [comment]
@greg0ire yes.
But I modified all form types in `form_types.xml`.

I tried and I can update this pull request to modify only `DateTimePickerType`, `DateRangePickerType` and `DateTimeRangePickerType` and it works without needed modification for `sonata.core.date.moment_format_converter`.

Let me know if I should update to this.

File: src/CoreBundle/Resources/config/form_types.xml [hunk 1]
```diff
@@ -23,12 +23,12 @@
             <tag name="form.type" alias="sonata_type_datetime_range"/>
             <argument type="service" id="translator"/>
         </service>
-        <service id="sonata.core.form.type.date_picker" class="Sonata\CoreBundle\Form\Type\DatePickerType" public="true">
+        <service id="sonata.core.form.type.date_picker" class="Sonata\Form\Type\DatePickerType" public="true">
```
Comment (core23) [inline src/CoreBundle/Resources/config/form_types.xml]: I'm not sure if we should do this. If someone want's to use the old form type, there could be problems

**Comment by core23** [comment]
Not after merging https://github.com/sonata-project/SonataCoreBundle/pull/629

**Pull request closed without merging**